dev-server/.eslintrc

```diff
@@ -0,0 +1,11 @@
+{
+  "env": {
+    "node": true,
+  },
+  "parserOptions": {
+    "sourceType": "script"
+  },
+  "rules": {
+    "@typescript-eslint/no-var-requires": "off",
```

Comment:
Turn off a typescript-eslint rule that bans CommonJS imports (`var foo = require(...)`).